argus/resources/windows/test_wmi.ps1

```diff
@@ -0,0 +1,19 @@
+#ps1_sysnative
+$ErrorOccured = $false
+
+try 
+{ 
+   $ErrorActionPreference = 'Stop'
+   (Get-WmiObject Win32_OperatingSystem).OperatingSystemSKU -notmatch 0
+   # Checks if its a valid Windows version
+}
+catch
+{
+   Set-Content 'c:\\Scripts\\wmi_log.txt' 'F'
```

Comment:
Use full parameter names. It should be: `Set-Content -Path C:\Scripts\wmi_log.txt -Value 'F'`